tests/jballerina-bstring-unit-test/src/test/resources/test-src/types/decimal/decimal_value_negative_literal.bal

```diff
@@ -19,6 +19,8 @@ function testInvlaidDecimalValue() {
     // This is to test a syntax issue.
     decimal a = 12.3f;
     decimal b = 0x4fp1;
+    decimal c = 0x12bc;
+    decimal d = -0x12b;
```

Comment:
Shall we add tests for `0X` and `-0X` too?

Reply:
Done